### PR TITLE
Backport #30079 to release 3 4

### PR DESCRIPTION
**Pull request by m-kuhn**
See #30079

### CODE REVIEW

File: src/gui/qgsmaptoolcapture.cpp [hunk 1]
```diff
@@ -30,6 +30,8 @@
 #include "qgsvectorlayer.h"
 #include "qgsvertexmarker.h"
 #include "qgssettings.h"
+#include "qgsapplication.h"
```
Comment (luipir) [inline src/gui/qgsmaptoolcapture.cpp]: this line in not present in master! is it necessary? seems not related with the added line in this file

Comment (m-kuhn) [reply src/gui/qgsmaptoolcapture.cpp]:
```suggestion
```

Comment (m-kuhn) [reply src/gui/qgsmaptoolcapture.cpp]: Thanks it was the result of a merge conflict

**Review by luipir — APPROVED** [review]
